students/alheadstrong/lesson_9/mailroom.py

```diff
@@ -0,0 +1,204 @@
+import os
+import sys
+
+
+class Donor(object):
+    def __init__(self, name, donation_list):
+        self.name = name
+        self.donations = donation_list
+        self._donation_total = sum(donation_list)
+        self._donation_average = self._donation_total / len(self.donations)
+
+    @property
+    def donation_total(self):
+        self._donation_total = sum(self.donations)
+        return self._donation_total
+
+    @property
+    def donation_average(self):
+        self._donation_total = self.donation_total
+        self._donation_average = self._donation_total/len(self.donations)
+        return self._donation_average
```

Comment:
you don't need the extra hidden variables, can be simply:
```
@property
    def donation_total(self):
    return sum(self.donations)
```
same for the average and number of donations